spec/features/hub/clients_searching_sorting_and_filtering_spec.rb

```diff
@@ -37,6 +38,16 @@
         expect(page.all('.client-row')[0]).to have_text(zach_prep_ready_for_call.preferred_name)
         click_button "Clear filters"
 
+        within ".filter-form" do
+          select "Alan's Org", from: "vita_partner_id"
+          click_button "Apply"
+          expect(page).to have_select("vita_partner_id", selected: "Alan's Org")
+        end
+
+        expect(page.all('.client-row').length).to eq 1
+        # expect(page.all('.client-row')[0]).to have_text alan_intake_in_progress.preferred_name
```

Comment:
Minor detail: it looks like this was intended to uncommented or deleted. Uncommented makes most sense to me.